CruzHacks/cruzhacks-2021-website
# Fixing Auto-Deploy Bug

Problem & Solution
=======
We need to build two targets for our CLI to work. The first target, `cruzhacks-2021-website`, will contain the production build while the second target, `cruzhacks-2021-website-test`, will contain the testing/staging site where we will be incrementally pushing up our changes. 

I have also added the ability for the GithubActions workflow to happen in `production` branch and PR against `production`

Steps to Verify:
----------------
I ran the command in the GithubActions workflow. Ensure that the site [https://cruzhacks-2021-website-test.web.app/](https://cruzhacks-2021-website-test.web.app/) has the build & if you have access to Firebase console, make sure that the most recent deploy was made by our dev@cruzhacks account.
